### PR TITLE
Bake refactor #2

### PR DESCRIPTION
**Pull request by rsamigullin**

### CODE REVIEW

File: src/helm-util.ts [hunk 1]
```diff
@@ -5,52 +5,20 @@ import * as os from 'os';
...
 import * as toolCache from '@actions/tool-cache';
 import * as core from '@actions/core';
 import * as io from '@actions/io';
 
 const helmToolName = 'helm';
-const stableHelmVersion = 'v2.14.1';
-const helmLatestReleaseUrl  = 'https://api.github.com/repos/helm/helm/releases/latest';
+
 
 export interface NameValuePair {
```
Comment (OliverMKing) [inline src/helm-util.ts:17]: where is this used?

File: src/helm-util.ts [hunk 2]
```diff
@@ -5,52 +5,20 @@ import * as os from 'os';
...
+import { getExecutableExtension, isEqual, LATEST, setCachedToolPath, getStableVerison } from "./utilities"
 
 import * as toolCache from '@actions/tool-cache';
 import * as core from '@actions/core';
 import * as io from '@actions/io';
 
 const helmToolName = 'helm';
-const stableHelmVersion = 'v2.14.1';
-const helmLatestReleaseUrl  = 'https://api.github.com/repos/helm/helm/releases/latest';
+
```
Comment (OliverMKing) [inline src/helm-util.ts:16]: have an extra space here

**Review by OliverMKing — APPROVED** [review]

**Comment by OliverMKing** [comment]
Nice changes :)

File: lib/kubectl-util.js [hunk 3]
```diff
@@ -21,48 +20,14 @@ const toolCache = require("@actions/tool-cache");
...
-            core.warning(util.format("Failed to read latest kubectl version from stable.txt. From URL %s. Using default stable version %s", stableVersionUrl, stableKubectlVersion));
-            return stableKubectlVersion;
-        });
-    });
-}
-exports.getStableKubectlVersion = getStableKubectlVersion;
 function downloadKubectl(version) {
     return __awaiter(this, void 0, void 0, function* () {
+        if (!version) {
+            version = yield (0, utilities_1.getStableVerison)(kubectlToolName);
```
Comment (bfoley13) [inline lib/kubectl-util.js:26]: can we use something like utilities/utils instead of utilities_1

File: lib/kompose-util.js [hunk 4]
```diff
@@ -27,18 +26,18 @@ function getKomposePath() {
         let komposePath = "";
         const version = core.getInput('kompose-version', { required: false });
         if (version) {
-            if (!!version && version != "latest") {
-                komposePath = toolCache.find('kompose', version);
+            if (!!version && version != utilities_1.LATEST) {
```
Comment (bfoley13) [inline lib/kompose-util.js:29]: same here with utilities_1

File: src/helm-util.ts (hunk 1 above)
Comment (rsamigullin) [reply src/helm-util.ts:17]: @OliverMKing it is used in "src/run.ts"

**Review by bfoley13 — APPROVED** [review]

**Review by davidgamero — APPROVED** [review]

**Pull request merged**